django/db/models/base.py

```diff
@@ -660,6 +660,34 @@ def _set_pk_val(self, value):
                 setattr(self, parent_link.target_field.attname, value)
         return setattr(self, self._meta.pk.attname, value)
 
+    @classmethod
+    def _validate_force_insert(cls, force_insert):
+        if not force_insert:
+            return ()
+
+        if isinstance(force_insert, bool):
+            return (cls,)
+
+        if isinstance(force_insert, ModelBase):
+            force_insert = (force_insert,)
+
+        if not isinstance(force_insert, tuple):
+            raise TypeError("force_insert must be a bool or tuple")
+
+        if not all(isinstance(item, ModelBase) for item in force_insert):
+            raise TypeError(
+                "All the items passed in force_insert must be a "
+                "subclass of models.Model"
+            )
+
+        if not all(issubclass(cls, item) for item in force_insert):
+            raise TypeError(
+                "The current model must be a subclass of"
+                "all the items passed in force_insert"
+            )
```

Comment:
Feel like providing more details about which member is not a subclass of `cls` might be more helpful

```suggestion
        for base in force_insert:
            if not issubclass(cls, base):
                raise TypeError(
                    f"Invalid force_insert member. {base!r} must be a base of {cls!r}."
                )
```